package.json

```diff
@@ -76,7 +76,7 @@
     "co": "^4.6.0",
     "co-express": "^1.2.1",
     "coffee-loader": "^0.7.2",
-    "coffee-script": "1.9.x",
```

Comment:
Why did we change the coffee script version?

Reply:
I believe it was due to the way I was installing coffee-script, I've updated so that this does not get changed.